internal/trigger/gaussian/gaussian_rate.go

```diff
@@ -15,11 +16,12 @@ import (
 
 func GaussianRate() api.Builder {
 	flags := pflag.NewFlagSet("gaussian", pflag.ContinueOnError)
-	flags.Float64("volume", 24*60*60, "The desired volume to be achieved with the calculated load profile.")
+	flags.Float64("volume", 0, "The desired volume to be achieved with the calculated load profile.")
```

Comment:
I haven't find a better way to mark this as optional.

Reply:
I think it would be better to leave the default value, but just state that it will be ignored if `--peak-rate` is provided. E.g:
```suggestion
	flags.Float64("volume", 24*60*60, "The desired volume to be achieved with the calculated load profile. Will be ignored if --peak-rate is also provided.")
```